packages/toolpad-app/src/toolpad/AppEditor/GlobalScopeExplorer.tsx

```diff
@@ -39,6 +39,10 @@ function groupScopeMeta(value: Record<string, unknown>, meta: GlobalScopeMeta):
       displayName: 'Queries',
       items: [],
     },
+    hidden: {
```

Comment:
Introduce hidden bindings, as there are bindings created for some nodes that I didn't want to show in others.